xrdp/xrdp.ini.in

```diff
@@ -229,12 +229,22 @@ password=ask
 #delay_ms=2000
 
 [neutrinordp-any]
-name=neutrinordp-any
+name=RDP Proxy (NeutrinoRDP)
+; To use this section, you should build xrdp with configure option
+; --enable-neutrinordp.
 lib=libxrdpneutrinordp.@lib_extension@
 ip=ask
 port=ask3389
 username=ask
 password=ask
+; You can Uncomment the following line to enable PAM authentication for Proxy 
```

Comment:
I'm not really happy with the second bit of this comment as it applies to all of the stanzas, and not just neutrinordp-any, which is probably the least used of all of them. Also you haven't mentioned `ls_height`.

What I suggest you do is leave this comment as:-
```
; Uncomment the following lines to enable PAM authentication for proxy
; connections
```

Then add the following under `; width and height of login screen`:-

```
;
; The default height allows for about 5 fields to be comfortably displayed
; above the buttons at the bottom. To display more fields, make <ls_height>
; larger, and also increase <ls_btn_ok_y_pos> and <ls_btn_cancel_y_pos>
; below
```

Come back to me if you've got a better idea.

Reply:
Thank you very much.

My writing was not good.
I think your comment is better.
I will fix the description.

Reply:
Fixed in #1f09ad9.